### PR TITLE
filter list of content type in workflow profiles

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: WorkflowAdminBundle/EventSubscriber/AddGroupWorkflowProfileSubscriber.php [hunk 1]
```diff
@@ -26,19 +27,22 @@ class AddGroupWorkflowProfileSubscriber implements EventSubscriberInterface
...
         WorkflowProfileRepositoryInterface $workflowProfileRepository,
         ContentTypeRepositoryInterface $contentTypeRepository,
+        SiteRepositoryInterface $siteRepository,
         DataTransformerInterface $workflowProfileCollectionTransformer,
         CurrentSiteIdInterface $contextManager,
         TranslatorInterface $translator
     ) {
         $this->workflowProfileRepository = $workflowProfileRepository;
         $this->contentTypeRepository = $contentTypeRepository;
+        $this->siteRepository = $siteRepository;
```
Comment (alavieille) [inline WorkflowAdminBundle/EventSubscriber/AddGroupWorkflowProfileSubscriber.php]: add it in class property

File: WorkflowAdminBundle/EventSubscriber/AddGroupWorkflowProfileSubscriber.php [hunk 2]
```diff
@@ -58,9 +62,14 @@ public function addWorkflowProfile(GroupFormEvent $event)
         }
 
         $configuration['default']['column'][NodeInterface::ENTITY_TYPE] = $this->translator->trans('open_orchestra_workflow_admin.profile.page');
-        $contentTypes = $this->contentTypeRepository->findAllNotDeletedInLastVersion();
-        foreach ($contentTypes as $contentType) {
-            $configuration['default']['column'][$contentType->getContentTypeId()] = $contentType->getName($this->contextManager->getCurrentLocale());
+
+        $siteId = $this->contextManager->getCurrentSiteId();
```
Comment (alavieille) [inline WorkflowAdminBundle/EventSubscriber/AddGroupWorkflowProfileSubscriber.php]: it's not the current site

**Comment by itkg-nanne** [comment]
update done @alavieille

File: WorkflowAdminBundle/EventSubscriber/AddGroupWorkflowProfileSubscriber.php [hunk 3]
```diff
@@ -51,19 +56,24 @@ public function __construct(
      */
     public function addWorkflowProfile(GroupFormEvent $event)
     {
+        $builder = $event->getBuilder();
+        $siteId = $this->contextManager->getCurrentSiteId();
+        $group = $builder->getData();
+        $site = ($group instanceof GroupInterface) ? $group->getSite() : $this->siteRepository->findOneBySiteId($siteId);
```
Comment (alavieille) [inline WorkflowAdminBundle/EventSubscriber/AddGroupWorkflowProfileSubscriber.php]: if $group->getSite is null no take currentSite

**Comment by itkg-nanne** [comment]
update done @alavieille

**Comment by itkg-nanne** [comment]
up @alavieille

File: WorkflowAdminBundle/EventSubscriber/AddGroupWorkflowProfileSubscriber.php [hunk 4]
```diff
@@ -15,7 +16,7 @@
 /**
  * Class AddGroupWorkflowProfileSubscriber
  */
-class AddGroupWorkflowProfileSubscriber implements EventSubscriberInterface
+    class AddGroupWorkflowProfileSubscriber implements EventSubscriberInterface
```
Comment (alavieille) [inline WorkflowAdminBundle/EventSubscriber/AddGroupWorkflowProfileSubscriber.php]: remove indent

**Comment by alavieille** [comment]
:+1: else indent

**Pull request merged**